docs/reference/remap/errors/601_invalid_timestamp.cue

```diff
@@ -0,0 +1,28 @@
+package metadata
+
+remap: errors: "601": {
+	title:       "Invalid timestamp"
+	description: """
+		The provided [timestamp literal](\(urls.vrl_expressions)#\(remap.literals.timestamp.anchor)) is properly
+		formed (i.e. it uses `t'...'` syntax) but the timestamp doesn't adhere to [RFC 3339](\(urls.rfc_3339)) format.
+		"""
+
+	rationale: "Invalid timestamps don't compile."
+
+	resolution: """
+		Bring the timestamp in conformance with [RFC 3339](\(urls.rfc_3339)) format.
+		"""
+
+	examples: [
+		{
+			"title": "\(title) (parsing)"
+			source: #"""
+				parse_timestamp!("next Tuesday", format: "%v %R")
+				"""#
+			diff: #"""
+				- 	parse_timestamp!("next Tuesday", format: "%v %R")
+				+# 	parse_timestamp!("10-Oct-2020 16:00", format: "%v %R")
+				"""#
+		},
+	]
```

Comment:
This example shows something different from what the `601` error is about. The description does describe it correctly, though.

It's when you do `t'nope'` that you'll get this error, not when you incorrectly call a function such as `parse_timestamp`.

Reply:
Ah, that's a mistaken copy/paste. Will fix.